smart_campus/app/views.py

```diff
@@ -191,6 +186,39 @@ def station_list_page(request):
 
 
 @login_required
+def station_list_by_category_page(request, pk):
+    category = get_object_or_404(StationCategory, pk=pk)
+    if request.user.can(Permission.ADMIN):
+        stations = Station.objects.filter(category=category)
+    else:
+        stations = Station.objects.filter(owner_group=request.user.group, category=category)
+
+    station_data = [
+        {
+            'id': station.id,
+            'name': station.name,
+            'primary_image': StationImage.objects.filter(
+                    station=station,
+                    is_primary=True
+                ).first(),
```

Comment:
Four fewer spaces from 201~203

Reply:
Okay, Fixed it.
I think it was result from the cherry-pick process I didn't check out.